lib/platform/github/index.ts

```diff
@@ -115,12 +115,6 @@ export async function initPlatform({
     renovateUsername,
   };
 
-  // Generic github hostRule that per default all datasources using github api are enabled
-  const genericGithubHostRule = {
-    matchHost: parseUrl(defaults.endpoint).hostname,
-    token,
-  };
-  hostRules.add(genericGithubHostRule);
```

Comment:
We already addind one here:
https://github.com/renovatebot/renovate/blob/fb75b2f4dc8e72bc5457800c386310ad020da403/lib/platform/index.ts#L58-L70